java/alizer-api/src/main/java/com/redhat/devtools/alizer/api/Component.java

```diff
@@ -10,25 +10,35 @@
  ******************************************************************************/
 package com.redhat.devtools.alizer.api;
 
+import java.io.File;
 import java.nio.file.Path;
 import java.util.List;
 
 public class Component {
 
     private Path path;
+    private File config;
```

Comment:
Not sure about it because i try to use them logically for what they should be representing. Here, `config` is the configuration file (if any) of the component (e.g pom.xml, build.gradle ....) so it makes sense to being handled as a file. Opposite there is `path` which is the location where the component (root path) is stored.